client/main.lua

```diff
@@ -991,13 +995,14 @@ Citizen.CreateThread(function()
 				LastStation             = currentStation
 				LastPart                = currentPart
 				LastPartNum             = currentPartNum
+				LastZone				= nil
 
-				TriggerEvent('esx_mechanicjob:hasEnteredvehMarker', currentStation, currentPart, currentPartNum)
+				TriggerEvent('esx_mechanicjob:hasEnteredMarker', LastZone, currentStation, currentPart, currentPartNum)
 			end
 
 			if not hasExited and not isInMarker and HasAlreadyEnteredvehMarker then
 				HasAlreadyEnteredvehMarker = false
-				TriggerEvent('esx_mechanicjob:hasExitedvehMarker', LastStation, LastPart, LastPartNum)
+				TriggerEvent('esx_mechanicjob:hasExitedMarker', LastZone, LastStation, LastPart, LastPartNum)
```

Comment:
Line 812 will throw an error because the AddEventHandler only accepts single arguments (zone).